engine/src/test/java/io/zeebe/engine/processing/streamprocessor/ProcessExecutionRandomizedPropertyTest.java

```diff
@@ -64,6 +69,7 @@ public TestDataRecord getDataRecord() {
   @Test
   public void shouldExecuteProcessToEnd() {
     final BpmnModelInstance model = record.getBpmnModel();
+    LOG.error("Generated Model: {}", Bpmn.convertToString(model));
```

Comment:
Debugging left over? I guess the LOG in general as well.

Reply:
Ah yes sorry